Python/import.c

```diff
@@ -1146,8 +1146,9 @@ get_cached_m_dict(struct extensions_cache_value *value,
 }
 
 static void
-del_extensions_cache_value(struct extensions_cache_value *value)
+del_extensions_cache_value(void *raw)
```

Comment:
You can also update `_extensions_cache_init()` by removing the cast to `_Py_hashtable_destroy_func` now that it's no more used.